### PR TITLE
set etc/hosts for e2e to work with proper ui nginx config

### PR DESCRIPTION
**Pull request by pazembrz**
TEST

### CODE REVIEW

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -297,6 +306,9 @@ jobs:
           name: ui
           options: --mount type=bind,source=${{ github.workspace }}/ui/docker/nginx/config/local.conf,destination=/etc/nginx/conf.d/default.conf
 
+      - name: Set hosts on ui
+        run: docker exec ui sh -c 'echo -e "127.0.0.1 hep-web\n127.0.0.1 next-web\n127.0.0.1 record-editor" >> /etc/hosts'
```
Comment (benjamin-bergia) [inline .github/workflows/integration.yml:310]: This is not the correct way to modify the hosts file.
https://docs.docker.com/engine/reference/commandline/run/#add-entries-to-container-hosts-file---add-host

File: .github/workflows/integration.yml [hunk 2]
```diff
@@ -226,6 +226,15 @@ jobs:
         with:
           ref: ${{ env.REF }}
 
+      - name: Pull images
```
Comment (benjamin-bergia) [inline .github/workflows/integration.yml:229]: What's the difference between this and adding `&` in the action?

**Comment by benjamin-bergia** [comment]
All this changes so you don't need to add 
```yaml
network_mode: "host"
```
to the compose file?

File: .github/workflows/integration.yml (hunk 2 above)
Comment (pazembrz) [reply .github/workflows/integration.yml:229]: This pulls all images in parallel (for all actions). In actions you download only one image (for example `hep`)and then you run it, then you download next image *for example `next` this time) and then you run it. which might be significantly slower.

**Comment by pazembrz** [comment]
> All these changes so you don't need to add
> 
> ```yaml
> network_mode: "host"
> ```
> 
> to the compose file?

Yes, because on my machine it will break as I have already PostgreSQL and ES running for other service on my machine (on their native ports) And I would like not to fight with this every-time I want to run tests ;) Otherwise I could run everything naively.

File: .github/workflows/integration.yml (hunk 1 above)
Comment (pazembrz) [reply .github/workflows/integration.yml]: :+1: Thanks, I completely forgot about that.

**Review by benjamin-bergia — APPROVED** [review]

**Pull request merged**